content/contributing/project-information/working-groups.md

```diff
@@ -0,0 +1,65 @@
++++
+title = "Working Groups"
+insert_anchor_links = "right"
+[extra]
+weight = 1
++++
+
+Working groups are temporary community initiatives devoted to particularly, complex, large, or important tasks. Some examples include:
+
+- Revamping Bevy's color APIs
+- Replacing Bevy's render graph
+- Implementing relations
+- And drafting this Contributing Guide!
+
+:::info
+You can read more about how to find and join active working groups in the [How You Can Help](todo) section.
+:::
+
```

Comment:
Worth having an active and explicit call-out for onboarding newcomers. Something like

"If you are new to Bevy development and don't know what to contribute to, have a look at what working groups are available! Maybe one of them will grab your interest. Just join in!"

Reply:
Agreed.